File: Models/JsonViewModels/parkinginfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SmartParking.Admin.Models.JsonViewModels
{

    public class parkinginfo
    {
        public string index { get; set; }
        public string zone { get; set; }
        public string total { get; set; }
        public string free { get; set; }
        public string occupied { get; set; }
        public string paid { get; set; }

    }
}

File: wwwroot/js/parkinginfo.js
var parkinginfo = {
    table: null,
    loadingOverlay: null,
   

    Init(table, loadingOverlay) {
        this.table = table;
        this.loadingOverlay = loadingOverlay;
    },

   
    LoadData() {
        $(this.loadingOverlay).LoadingOverlay("show"); 
        spAPI.getParkingInfo(function (data) {
                        
            $(parkinginfo.table).dataTable({
                "aaData": data,
                "bDestroy": true,
                "bFilter": false,
                "bPaginate": false,
                "bInfo": false,
                "bLengthChange": false,
                dom: 'Bfrtip',
                buttons: [
                    { "extend": 'excel', "text": '<i class="fa fa-file-excel-o"></i>', "className": 'btn btn-success btn-xs-6' }
                ],
                "columns": [
                    { "data": "index" },
                    { "data": "zone" },
                    { "data": "total" },
                    { "data": "free" },
                    { "data": "occupied" },
                    { "data": "paid" }

                ]
            });
            $(parkinginfo.loadingOverlay).LoadingOverlay("hide", true); 

            //setTimeout(parkinginfo.LoadData(), 10000);
        });
    }
    
}


